docs/guides/guide-using-victoriametrics-to-monitor-vmware-vcenter.md

```diff
@@ -0,0 +1,617 @@
+END
+
+cat <<END >/etc/victoriametrics/vmagent/scrape.yml
+# Scrape config example
+#
+global:
+  scrape_interval: 60s
+  scrape_timeout: 30s
+
+scrape_configs:
+  - job_name: vmagent
+    static_configs:
+    - targets: ['127.0.0.1:8429']
+  - job_name: victoria-metrics
+    static_configs:
+    - targets: ['127.0.0.1:8428']
+END
+
+# Start VictoriaMetrics
+systemctl enable vmagent.service
+systemctl restart vmagent.service
+```
+
+vmagent can be run with flags or specify to use parameters from variables. For the second option `/etc/victoriametrics/vmagent/vmagent.conf` will look this:
+
+telegraf.yml
+
+Running from /etc/systemd/system/vmagent.service with [systemd daemon](https://ru.wikipedia.org/wiki/Systemd):
+
+## vmware_exporter
+
+To install vmware_exporter we will use pip.
+
+To install in a user environment run the following command:
+
+```bash
+pip install vmware_exporter
+```
+
+Create `config.yml` file with the next configuration:
+
+```bash
+cat <<END >config.yml
+default:
+    vsphere_host: "vcenter-host"
+    vsphere_user: "vmuser"
+    vsphere_password: "pswd"
+    ignore_ssl: True
+    specs_size: 5000
+    fetch_custom_attributes: True
+    fetch_tags: False
+    fetch_alarms: False
+    collect_only:
+        vms: True
+        vmguests: True
+        datastores: False
+        hosts: False
+        snapshots: True
+END
+```
+
+Run `vmware_exporter`:
+
+```bash
+vmware_exporter -c config.yml
+```
+
+Check that exporter show metrics by running curl query:
+
+```bash
+curl http://localhost:9272/metrics
+```
+
+Add a job to poll the exporter in vmagent with the next configuration:
+
+```bash
+ - job_name: vcenter
+    metric_relabel_configs:
+    - regex: ^([A-Z]):.*
+      replacement: $1
+      source_labels:
+      - partition
+      target_label: partition
+    - action: drop
+      regex: /target|/storage/archive
+      source_labels:
+      - partition
+    - action: labeldrop
+      regex: host_name|ds_name
+    static_configs:
+    - targets:
+      - exporters-host:9272
+```
+
+When polling the exporter the relabel rules will be applied to all received metrics namely:
+the partition names of windows OS disks will have colons removed otherwise they will have to be escaped in queries;
+all metrics with partitions in linux `/target` or `/storage/archive` will be removed because there is no sense to collect them;
+remove `host_name` and `ds_name` labels to reduce the number of time series a.k.a. [churn rate](https://docs.victoriametrics.com/FAQ.html#what-is-high-churn-rate).
+
+## telegraf
+
+> The polling interval is specified in each configuration file or globally. It makes sense to make a lot of files because if the plugin does not have time to complete the work in the specified interval the data on all devices will be lost. When a problem occurs, the loss of monitoring data makes monitoring pointless. 
```

Comment:
> It makes sense to make a lot of files because if the plugin does not have time to complete the work in the specified interval the data on all devices will be lost.

I didn't get it.